### DISCUSSION TITLE
Maddy mail server how to specify the dkim selector?

### DISCUSSION THREAD

**Discussion by wangyu1314** [emersion/maddy#605] [General]
Maddy mail server how to specify the dkim selector? For example, I want to use my habit of 20230713 instead of default, please advise, thanks.

**Comment by jimstorch** [comment]
I'm just dipping my toe into Maddy, but I _think_ you want to change the **dkim** line in `maddy.conf` under the **submission** section:

```
default_destination {
    modify {
        dkim $(primary_domain) $(local_domains) default
     }
```
Try changing '_default_' to '_20230713_'.

**Reply by wangyu1314** [reply]
Thank you, I changed it, but it didn't work, and even without changing the slector, the windows client can connect to send mail, but the PHP forum script can't connect with the same username and password, so I fell back to postfix dovecot rspamd.  All problems solved.
